src/routes/tftRoutes.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express'
+
+function getTFTRoutes(){
+    const router = express.Router()
+    router.get('/summoner/:summonerName', getSummonerHandler)
+    return router
+}
+
+async function getSummonerHandler(req, res){
+    res.send("Welcome " + req.params.summonerName + "!!")
```

Comment:
This is one way to do this. The better/newer/more modern way to do this is to use something called interpolation. It looks like this, but we'll do this as part of the next phase:

res.send(`Welcome ${req.params.summonerName}!!`)